niflow_manager/cli/tests/test_main.py

```diff
@@ -11,8 +12,32 @@ def test_utility_options(option):
 
 
 @pytest.mark.parametrize("command", ["test", "install", "build"])
-def test_unimplemented(command):
+def test_commads_help(command):
     runner = CliRunner()
-    result = runner.invoke(main, [command])
+    result = runner.invoke(main, [command, "--help"])
     assert result.exit_code == 0
-    assert "not yet implemented" in result.stdout
+
+
+def test_init_build_python():
+    """ initilize a niflow for python and build a container using the default spec"""
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        res_init = runner.invoke(
+            main, ["init", "--language", "python", "wf"], "\n".join(["org", "wf", "y"])
+        )
+        niflow_path = Path.cwd() / "niflow-org-wf"
+        spec_path = niflow_path / "spec.yml"
+        assert res_init.exit_code == 0
+        assert spec_path.exists()
+
+        # uncommenting the spec file
+        with spec_path.open() as f:
+            spec_orig = f.readlines()
+        with spec_path.open(mode="w") as f:
+            for line in spec_orig:
+                # removing afni and fsl installation to shorten the test
+                if not ("fsl" in line or "afni" in line or "version" in line):
```

Comment:
why not change the test?

Reply:
?

Reply:
you wrote in the comment before

> # removing afni and fsl installation to shorten the test

if that is necessary, why not change the test to a shorter test.

Reply:
i'd have to change the `template` (this test just calls `init` and this create the spec from the templates), I could remove afni and fsl from the template, was just thinking that can be nice as an easy to see example

Reply:
i think examples can be different from tests in the repo. and it would be good to point to some examples in the docs, but the baseline operation should be minimal while covering all aspects of the spec.

Reply:
wanted to use the template here, but now it works without any changes to spec